Snippets/NHibernate/NHibernate_6/Usage.cs

```diff
@@ -94,7 +94,7 @@ void CustomCommonConfigurationWarning(BusConfiguration busConfiguration)
             }
         };
 
-        var persistence = busConfiguration.UsePersistence<NHibernatePersistence, StorageType.GatewayDeduplication>();
+        var persistence = busConfiguration.UsePersistence<NHibernatePersistence, StorageType.Subscriptions>();
```

Comment:
This snippet was using Gateway as a "for example" of overriding a specific type - Gateway wasn't important. So I changed the text to use Subscriptions (since splitting Saga/Outbox isn't appropriate) and changed the snippet on ALL versions.